modules/sandboxed-containers-deploying-workloads-with-kata-runtime-cli.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-cli_{context}"]
+= Deploying a workload in a sandboxed container using the CLI
+
+{sandboxed-containers-first} installs Kata as a secondary, optional runtime on your cluster, and not as the primary runtime.
+
+To deploy a pod-templated workload in a sandboxed container, you must add `kata` as the `runtimeClassName` to the workload YAML file.
```

Comment:
@sagidlow - so, considering a sandboxed container is an industry term, regardless of the product title, and since I was just referring to the item, not the product, I actually don't think this needs a variable. We use the term container all over the OCP help, and it isn't a variable. Would we consider this scenario to be different?